bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -797,8 +838,9 @@ func (r *KubeadmConfigReconciler) storeBootstrapData(ctx context.Context, scope
 		}
 		r.Log.Info("bootstrap data secret for KubeadmConfig already exists", "secret", secret.Name, "KubeadmConfig", scope.Config.Name)
 	}
-
+	time.Sleep(10 * time.Second)
```

Comment:
left over from debugging? seems odd to have a 10 second sleep here.

Reply:
ops...